docs/en/setup/configuration/profiling.md

```diff
@@ -106,4 +106,10 @@ Based on the above two data types, the following metrics are provided.
 
 | Name         | Type  | Unit        | Description                |
 |--------------|-------|-------------|----------------------------|
-| slow_traces  | TopN  | millisecond | The Top N slow trace(id)s  |
\ No newline at end of file
+| slow_traces  | TopN  | millisecond | The Top N slow trace(id)s  |
+
+##### Span Attached Event
+| Name               | Description                                                                                  |
+|--------------------|----------------------------------------------------------------------------------------------|
+| http-full-request  | Complete information about the HTTP request, it's only uploads when it matches slow traces.  |
+| http-full-response | Complete information about the HTTP response, it's only uploads when it matches slow traces. |
```

Comment:
For slow trace only? I think we have `slow`, `4xx`, `5xx` policies?

Reply:
For this PR, I only implemented the slow policies. I will create other PR to implement them all(contains protocols update).